deploy/template.yaml

```diff
@@ -422,9 +422,9 @@ Resources:
               awslogs-group: !Ref ECSAccessLogsGroup
               awslogs-region: !Sub ${AWS::Region}
               awslogs-stream-prefix: !Sub ${AWS::StackName}-${Environment}
-      Cpu: "1024"
+      Cpu: "256"
```

Comment:
I understand why we want to reduce these for cost but these memory values are high enough to make me think we need that much memory to complete a journey...Can we deploy a FE instance to ECS and confirm this is non-breaking